Data/TypeDocumentation.cs

```diff
@@ -183,13 +183,19 @@ void XmlToHtml (XElement x, TextWriter w)
                     w.Write (x.Attribute ("name")?.Value ?? "");
                     w.Write ("</span>");
                     break;
+                case "seealso":
                 case "see": {
                         var cref = x.Attribute ("cref");
                         if (cref != null && cref.Value.Length > 2) {
                             WriteMemberLinkHtml (cref.Value, w);
                         }
                     }
                     break;
+                case "typeparamref":
+                    w.Write ("<span class=\"inline-code c-tr\">");
+                    w.Write (x.Attribute ("name")?.Value ?? "");
```

Comment:
The request in https://github.com/praeclarum/FuGetGallery/pull/45#discussion_r213840926 would probably apply here as well.